.github/workflows/linting.yml

```diff
@@ -4,21 +4,11 @@ jobs:
   clang-format:
     name: clang-format style
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        path:
-          - "isobus"
-          - "socket_can"
-          - "examples"
```

Comment:
We may want to find some way to still exclude some of the 3rd party driver files, as their licenses may not allow us to modify them, such as if they are LGPL licensed

Reply:
Hmm, I see. Do we have any that should be excluded right now? I think we can use `exclude-regex` option for this? E.g:
```yml
# Some files can be excluded from the clang-format style check, like 3rd party driver files,
# as their licenses may not allow us to modify them, such as if they are LGPL licensed.
# The exclude-regex option can be used to exclude these files from the style check:
exclude-regex: ^(PCANBasic\.h)$
```
Or is there something else you had in mind?

Reply:
I have one on a branch here https://github.com/ad3154/ISO11783-CAN-Stack/blob/feature/innomaker-usb2can/hardware_integration/include/isobus/hardware_integration/libusb.h but nothing to worry about in main, as long as there's a way to do it I'm content 😁

Reply:
Hahah, all good, I've added the file to exclusion for you already (in the hope you aren't gonna rename it haha). You can easily append multiple files by using the `|` OR in the exclude regex filter 😄